gcp/src/test/java/org/apache/iceberg/gcp/gcs/GCSFileIOTest.java

```diff
@@ -20,14 +20,20 @@
 
 import static java.lang.String.format;
 import static org.assertj.core.api.Assertions.assertThat;
+import static org.mockito.Mockito.any;
+import static org.mockito.Mockito.doAnswer;
+import static org.mockito.Mockito.spy;
 
+import com.google.api.client.util.Lists;
```

Comment:
does this have to be a List from that particular API?

Reply:
No it is just any list, I updated this to use the Guava utility